modules/client-configuration/pages/autoinst-unattended.adoc

```diff
@@ -0,0 +1,62 @@
+The {productname} API allows you to create profiles for systems as if they were collected by the bare metal functionality.
+For information about declaring systems with the API, see xref:client-configuration:autoinst-unattended.adoc#create-system-profile[Create a System Profile Manually].
+
+
+[[bare-metal]]
+== Bare Metal Provisioning
+
+When you have the bare metal provisioning option enabled, any bare metal client connected to the {productname} network is automatically added to the organization as soon as it is powered on.
+When this operation is complete, the client is shut down, and it appears in the [guimenu]``Systems`` list, ready to be installed.
+
+
+
+.Procedure: Provisioning Bare Metal Clients
+. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Bare-metal systems].
+. Click btn:[Enable adding to this organization].
+. In the {productname} {webui}, navigate to [guimenu]``Systems``, select the client to provision, and go to the menu:Provisioning[Autoinstallation] tab.
+. Select the {ay} profile to use, and click btn:[Schedule Autoinstallation and Finish] to start the autoinstallation.
+
+New bare metal clients are added to the organization that belongs to the administrator who enabled the bare metal feature.
+To change the organization new clients are added to, disable the bare metal feature, log in as the administrator of the new organization, and then re-enable the feature.
+You can migrate already registered systems to other organizations if required, using the [guilabel]``Migrate`` tab.
+
+You can use the system set manager (SSM) with bare metal clients.
+However, not all SSM features are available for bare metal clients, because they do not yet have an operating system installed.
+This also applies to mixed sets that include bare metal systems.
+All features become available to the set when all the clients in the set have been provisioned.
+For more information on SSM, see xref:client-configuration:system-set-manager.adoc[].
+
+The server uses TFTP to provision the bare metal client, so the appropriate port and networks must be configured correctly in order for provisioning to be successful.
+
+
+[[create-system-profile]]
+== Create a System Profile Manually
+
+You can use an API call to declare a client's name, MAC address and fully qualified hostname.
+You can then schedule a the reinstallation from its page in {productname} {webui}.
+
+
+
+.Procedure: Reinstallation From a Manually Declared Profile
+
+. On the {productname} Server, at the command prompt, use the [systemitem]``system.createSystemProfile`` API call.
+  In this example, replace [literal]``<hw_addr>`` with a hardware address such as [literal]``00:25:22:71:e7:c6`` and [literal]``<name>`` with the name of your client:
++
+----
+$ spacecmd api -- --args '["systemname", {"hwAddress": "<hw_addr>", "hostname":"<name>"}]' system.createSystemProfile
```

Comment:
I need to check this again next week

Reply:
Waiting for your next remarks :smile_cat: .

Reply:
Ok, this API function cannot be used here. From my presentation:
```
createSystemProfile(name, data):

Very specific for SUSE Manager for Retail. Cannot create a PXE boot Record with Cobbler
```
So this is not working.
Instead use:
```
createSystemRecord (name, profile, kopts, comment, [iface])

* Creates a cobbler system record for a system that is not registered.
* Specify the profile and kernel options directly
* No System entry in SUSE Manager UI - it just exist in cobbler
* Usable for network booting or buildiso
```